themes/ddbasic/sass/components/panel/pane-taxonomy-menu.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?